starlark/testdata/time.star

```diff
@@ -0,0 +1,119 @@
+# Tests of time module.
+
+load('assert.star', 'assert')
+load('time.star', 'time')
+
+assert.eq(time.parse_time("2020-06-26T17:38:36Z"), time.from_timestamp(1593193116))
```

Comment:
Let's add some tests of the data types' basic operations:
```
type(t)
str(t)
{t: None} # hashable
```
Ditto for duration.